### PR TITLE
Update paths dropdown options 

### PR DESCRIPTION
**Pull request by liyiy**
## Changes

*Please describe.*  
*If this affects the frontend, include screenshots.*  

Basically this PR https://github.com/PostHog/posthog/pull/6163 without the weird merge issues I keep getting

## How did you test this code?

*Please describe.*

### CODE REVIEW

**Comment by neilkakkar** [comment]
Carrying over q from previous PR: Why do we have an interface for exclusions with the `AnyPropertyType` ? Sounds like it's pretty much useless, since we discard all information except the `value`.

Not for this PR, but let's make it a `string[]` interface instead?

File: frontend/src/scenes/paths/NewPaths.js [hunk 1]
```diff
@@ -366,16 +373,65 @@ export function NewPaths({ dashboardItemId = null, filters = null, color = 'whit
...
+                                                                    ])
+                                                                } else {
+                                                                    updateExclusions([{ value: pageUrl(pathItemCard) }])
+                                                                }
+                                                            }}
+                                                        >
+                                                            Exclude path item
+                                                        </Menu.Item>
+                                                        <Menu.Item
+                                                            onClick={() => copyToClipboard(pageUrl(pathItemCard, true))}
```
Comment (neilkakkar) [inline frontend/src/scenes/paths/NewPaths.js:425]: I think copyToClipboard should copy the full name? If you want to use it elsewhere, would be useless if it's truncated?

**Comment by neilkakkar** [comment]
Made some minor adjustments so I can merge your PR (so I can then merge mine, and you don't have to resolve merge conflicts again)

The master weirdness was the start point being borked in the funnel path PR. I've fixed that here as well^. 

Feel free to make another PR if you feel one of the adjustments I made is wrong :)

**Comment by neilkakkar** [comment]
> Carrying over q from previous PR: Why do we have an interface for exclusions with the AnyPropertyType ? Sounds like it's pretty much useless, since we discard all information except the value.

Okay, nevermind, this is because `PathItemSelector` expects that format

**Comment by EDsCODE** [comment]
Heads up there are still bugs here. Using the menu options on URL items don't work

**Comment by neilkakkar** [comment]
huh wut, did I bork something up O.o

**Comment by neilkakkar** [comment]
oh right the HTTPstrip is wrong :P

**Comment by liyiy** [comment]
ok it's being tracked here: https://github.com/PostHog/posthog/pull/6204

**Pull request merged**